spec/formatting.md

```diff
@@ -936,7 +936,7 @@ The _Default Bidi Strategy_ is defined as follows:
    right-to-left directionality, and to the message's directionality not being known.
 1. For each _expression_ `exp` in _pattern_:
    1. Let `fmt` be the formatted string representation of the _resolved value_ of `exp`.
-   1. Let `dir` be the directionality of `fmt`,
+   1. Let `dir` be the directionality of the _resolved value_ of `exp`,
       one of « `'LTR'`, `'RTL'`, `'unknown'` », with the same meanings as for `msgdir`.
```

Comment:
```suggestion
      1. Let `dir` be the directionality of `fmt`,
         one of « `'LTR'`, `'RTL'`, `'unknown'` », with the same meanings as for `msgdir`.
      > [!NOTE]
      > `dir` is usually computed from the locale of the formatter
      > or from other information, such as the directionality of the _resolved value_ of `exp`,
      > and not by introspecting the character sequence in `fmt`
```

Reply:
I didn't take your suggestion exactly as-is, but let me know what you think about [58afcfa](https://github.com/unicode-org/message-format-wg/pull/968/commits/58afcfa7a2acc42e54a9341db473e0e1b94e3391).